docs/website/docs/tutorial/load-data-from-an-api.md

```diff
@@ -247,11 +247,11 @@ print(load_info)
 ```
 <!--@@@DLT_SNIPPET_END incremental_merge-->
 
-Above we add `primary_key` argument to the `dlt.resource()` that tells `dlt` how to identify the issues in the database to find duplicates which content it will merge.
+Above we add a `primary_key` argument to the `dlt.resource()` that tells `dlt` how to identify the issues in the database to find duplicates which content it will merge.
 
 Note that we now track the `updated_at` field — so we filter in all issues **updated** since the last pipeline run (which also includes those newly created).
 
-Pay attention how we use **since** parameter from [GitHub API](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues)
+Pay attention to how we use the **since** parameter from [GitHub API](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues)
```

Comment:
```suggestion
Pay attention to how we use the `since` parameter from [GitHub API](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues)
```